android/src/main/java/com/RNAppleAuthentication/SignInWithAppleConfiguration.kt

```diff
@@ -17,6 +18,7 @@ data class SignInWithAppleConfiguration private constructor(
     private lateinit var state: String
     private lateinit var rawNonce: String
     private lateinit var nonce: String
+    private var fullScreen: Boolean = false
```

Comment:
documented as defaulting to true, and in other spots has nullish-coalescing default to true, but here is false default

```suggestion
    private var fullScreen: Boolean = true
```